Changed Mono to Xamarin.Android

TheCodeTraveler opened the pull request.

TheCodeTraveler committed:
Changed Mono to Xamarin.Android

guperrot commented on docs/sdk/push/migration/xamarin-android.md:
```diff
@@ -14,9 +14,9 @@ ms.tgt_pltfrm: android
 [!include[](introduction-android.md)]
 
-## 1. Mono version
+## 1. Xamarin.Android version
```
Target framework version (as per the UI).

TheCodeTraveler replied:
<img width="982" alt="image" src="https://user-images.githubusercontent.com/13558917/47235796-f724c900-d38e-11e8-8404-977edb1a12b0.png">
👍 Good call with `Target Framework`. That's what the IDE calls it.

guperrot commented on docs/sdk/push/migration/xamarin-android.md:
```diff
@@ -14,9 +14,9 @@ ms.tgt_pltfrm: android
+## 1. Xamarin.Android version
 
-If your Android project does not target Mono framework version 8.1 or higher, you will not be able to update the Push package.
+If your Android project does not target Xamarin.Android version 8.1 or higher, you will not be able to update the Push package.
```
If the target framework version of your Android application is lower than 8.1, you will not be able to update the Push package.

guperrot reviewed (commented):
Suggesting an alternative wording

TheCodeTraveler committed:
Update xamarin-android.md

Used Target framework to maintain consistency with the nomenclature used in the IDE
